compilers/concrete-optimizer/concrete-optimizer/src/optimization/dag/multi_parameters/analyze.rs

```diff
@@ -51,6 +51,7 @@ pub fn analyze(
     noise_config: &NoiseBoundConfig,
     p_cut: &Option<PrecisionCut>,
     default_partition: PartitionIndex,
+    allow_composition: bool,
```

Comment:
should be in noise_config, it could be added later to the other solvers, and when more complex composition will be added it will not change the function signature.